inspectit-oce-agent/README.md

```diff
@@ -0,0 +1,20 @@
+# inspectIT OCE - Agent
+
+This project basically contains two components: the actual Java agent which will be attached to your JVM and a "system test".

```

Comment:
I would formulate this better, there are not really 2 components, but maybe two responsibilities of this project or two goals.. Smth in this direction..